cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,41 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
+  const username = "dieromin";
+  const password = "#Fcdk1927";
+  const book = "Speaking JavaScript";
+  const description = "Like it or not, JavaScript is everywhere";
 
+  beforeEach(() => {
+  
+    cy.visit('/books');
```

Comment:
Maybe, you should use cy.visit('/login')

Reply:
Fixed